### PR TITLE
adding sample notebook for extracting slums from satellite imagery and detecting brick kilns sample.

### LINKED ISSUE #2457: OfflineContentManager export_items fails for "Tile layer (hosted)" map service
**Describe the bug**
When using the OfflineContentManager's export_items method with a "Tile layer (hosted)" map service, published from a Tile Package, it unexpectedly fails. (See error message below.)

**To Reproduce**
In order to reproduce the issue you can attempt to use export_items with a "Tile layer (hosted)" map service, published from a Tile package. It will result in the following error:

```python
C:\Users\knoop\AppData\Local\ESRI\conda\envs\work\Lib\site-packages\arcgis\apps\itemgraph\_migration.py:149: RuntimeWarning: Failed to export item 9ad056d339534fdb9c75fa354491f101 due to error: <MapRasterLayer url:"https://tiles.arcgis.com/tiles/4ezfu5dIwH83BUNL/arcgis/rest/services/Environs_Jerusalem_NLI/MapServer/0"> is not JSON serializable. Deleting folder and skipping...
  warnings.warn(
```

**Expected behavior**
I expect export_items to be able to successfully export a "Tile layer (hosted)" map service, including its Tile package.

**Platform (please complete the following information):**
Python API 2.4.2

### PR DESCRIPTION
**Pull request by sandeepgadhwal**
Fixes #2457 arcgis.learn - Slum Change Detection notebook sample.

added sample notebook for extracting slums from satellite imagery.

### CODE REVIEW

**Comment by AtmaMani** [comment]
@cyber-aman could you do the first round of reviews?

**Comment by rohitgeo** [comment]
@sandeepgadhwal the dashboard is not loading fully (and it was private):

image

**Comment by sandeepgadhwal** [comment]
> @sandeepgadhwal the dashboard is not loading fully (and it was private):
> 
> image

I cloned the dashboard but some layers and functionalities failed, I will try to publish here again.

**Comment by AtmaMani** [comment]
@sandeepgadhwal could you post a comment that its ready for review after you have reviewed and incorporated @cyber-aman's comments and the dashboard?

**Comment by AtmaMani** [comment]
@sandeepgadhwal is this ready for review?

**Comment by sandeepgadhwal** [comment]
@AtmaMani this can be review now.

**Comment by AtmaMani** [comment]
@sandeepgadhwal @cyber-aman Thanks for the sample and detailed reviews. It looks good except the brick klin sample has a high validation loss at 10 epochs. Can we improve it?

**Comment by sandeepgadhwal** [comment]
@AtmaMani the validation loss is going down continuously, it means the model is getting trained. We cannot compare the training loss and validation loss directly.

**Comment by AtmaMani** [comment]
@sandeepgadhwal agreed. I mistook validation loss as f1 score.
